tpl/skel/config/cloud.yml

```diff
@@ -1,11 +1,12 @@
 default: &default
-  region: "us-east1" # Note that region formats differ between AWS and GCP
-  zone: "us-east1-a"
+  # Note that region formats differ between AWS and GCP
+  region:             "us-east1"
+  zone:               "us-east1-a"
```

Comment:
those params should be in gcp
but why renaming bucket_name? 

did you get a conflict of some sort?